data/yam/agama/auto/default_tumbleweed.sh

```diff
@@ -1,7 +1,7 @@
 set -ex
```

Comment:
we don't use this, we can drop this file.

Reply:
removed. So later we won't add such test for profile file extension of sh?

Reply:
I don't think now it is even an option this kind of scripts, that was an ALP prototype.